example/redash_one_node/main.tf

```diff
@@ -1,27 +1,67 @@
 module "redash" {
-  source = "../../"
-  # postgres
-  postgres_service_name   = module.postgres.service_name
-  postgres_container_port = module.postgres.port
-  postgres_username       = module.postgres.username
-  postgres_password       = module.postgres.password
-
-  depends_on = [
-    module.postgres
-  ]
-}
+  source = "../.."
+  # nomad
+  nomad_datacenters = ["dc1"]
+  nomad_namespace   = "default"
 
-output "redash_username"{
-  description = "Redash admin username"
-  value       = module.redash.redash_admin_username
+  # redash
+  service         = "redash"
+  host            = "127.0.0.1"
+  port            = 5000
+  container_image = "redash/redash:9.0.0-beta.b42121"
```

Comment:
Dette har ikke så mye å si for kjøring i Vagrant. Men versjon 9 kommer vi nok ikke til å kjøre på F2 med det første. Kanskje like greit å sette `redash/redash` (versjon 8) her?

Reply:
Enig!

Er det dette vi skal benytte: `redash/redash:8.0.2.b37747` ?
https://hub.docker.com/r/redash/redash/tags?page=1&ordering=last_updated

Reply:
Jepp, det burde fungere fint. Når vi skal over til F2 er det et custom overbygg-image vi vil benytte som baserer seg på `latest`, men det burde ikke ha noe å si.